projects/serializers.py

```diff
@@ -40,15 +50,15 @@ def validate(self, data):
                     "message": "Budget ready cannot be greater than budget needed"
                 }
             )
-        industry = data.get('industry')
-        if industry:
-            industries = [industry[0] for industry in Project.INDUSTRY_CHOICES]
-            if industry not in industries:
-                raise serializers.ValidationError(
-                    {
-                        "status": "failed",
-                        "message": "Industry does not exist",
-                        "choices": industries
-                    }
-                )
+        # industry = data.get('industry')
+        # if industry:
+        #     industries = [industry[0] for industry in Project.INDUSTRY_CHOICES]
+        #     if industry not in industries:
+        #         raise serializers.ValidationError(
+        #             {
+        #                 "status": "failed",
+        #                 "message": "Industry does not exist",
+        #                 "choices": industries
+        #             }
+        #         )
         return data
```

Comment:
If this code we would'n use in future, maybe we can delete it? (commented code)

Reply:
Ok, thank you